docs/components/cursor.md

```diff
@@ -80,6 +80,7 @@ AFRAME.registerComponent('cursor-listener', {
 | downEvents  | Array of additional events on the entity to *listen* to for triggering `mousedown` (e.g., `triggerdown` for vive-controls).  | []                               |
 | fuse        | Whether cursor is fuse-based.                                                                                              | false on desktop, true on mobile |
 | fuseTimeout | How long to wait (in milliseconds) before triggering a fuse-based click event.                                             | 1500                             |
+| mode     | Camera (gaze) or mouse controlled. | gaze
```

Comment:
Maybe we have different expectations on what a flag means. When seeing `mouseMode: true` enabled many people will interpret it as `I can use the mouse cursor as well to select things` in addition to the normal behavior. A `mode` word indicates... modality :smile: We can replace `gaze` with `screen` \ `entity` to indicate where the cursor performs the raycasting from. A cleaner solution is probably adding two raycasters and, an objects property and a  `mouseMode` flag (false by default) so the modes are not mutually exclusive. The API would be then:
`<a-entity cursor="mouseMode: true; objects: clickable">` 
The cursor would operate in both modes simultaneously and use two raycasters: `raycaster__mouse` and `raycaster__entity`